Core/src/org/sleuthkit/autopsy/modules/dataSourceIntegrity/Bundle.properties

```diff
@@ -1,15 +1,19 @@
 OpenIDE-Module-Name=ewfVerify
-EwfVerifyIngestModule.moduleName.text=E01 Verifier
-EwfVerifyIngestModule.moduleDesc.text=Validates the integrity of E01 files.
+EwfVerifyIngestModule.moduleName.text=Data Source Integrity
```

Comment:
I would have thought these would get renamed the same way the module name and class file name got renamed. It might provide for better clarity as to what one is looking at in case they're not familiar with the history of the module.